resolwe_bio/processes/slamdunk/alleyoop_rates.py

```diff
@@ -0,0 +1,68 @@
+"""Run Alleyoop rates tool on Slamdunk results."""
+import os
+
+from plumbum import TEE
+
+from resolwe.process import Cmd, DataField, FileField, Process, StringField
+
+
+class AlleyoopRates(Process):
+    """Run Alleyoop rates."""
+
+    slug = 'alleyoop-rates'
+    process_type = 'data:alleyoop:rates'
+    name = 'Alleyoop rates'
+    requirements = {
+        'expression-engine': 'jinja',
+        'executor': {
+            'docker': {
+                'image': 'resolwebio/slamdunk:1.0.0'
+            },
+        },
+        'resources': {
+            'cores': 1,
+            'memory': 16384,
+        },
+    }
+    entity = {
+        'type': 'sample',
+    }
+    data_name = '{{ slamdunk|sample_name|default("?") }}'
```

Comment:
Please add `category = 'Slamdunk'` to Slamdunk/Alleyoop processes.

Reply:
This has been resolved, should be marked as such.